### ISSUE TITLE
Can't read sheet if there is no protected range

### ISSUE THREAD

**Issue by uhhiitsphilia** [tidyverse/googlesheets4#185]
Hi! I tried to read a sheet but faced this error:

```r
> ss <- read_sheet("1AFl2QYzMOI55C_XXXXXXXXXXXXXXXX")
Error in `*tmp*`[[jj]] : subscript out of bounds
```
I later found that the cause of the error was these few lines of code in R/schema_Spreadsheet.R
```r
protected_ranges <- map(sheets, "protectedRanges")
protected_ranges <- purrr::flatten(protected_ranges)
protected_ranges <- map(protected_ranges, ~ new("ProtectedRange", !!!.x))
protected_ranges <- map(protected_ranges, as_tibble)
out$protected_ranges <- do.call(rbind, protected_ranges)
```
Once I added an unnecessary protected range in the actual google sheet as a temporary workaround, I didn't have any issues anymore.

**Comment by jennybc** [comment]
[How to create a googlesheets4 reprex](https://googlesheets4.tidyverse.org/articles/articles/googlesheets4-reprex.html)

**Comment by jennybc** [comment]
I have probably fixed this since you posted (in 7e68cdafbb9f3149fdbcc6b5d09968f25c43c0ee). I'm going to close on that assumption, but check back in if the problem persists after updating from `HEAD`.
